launchdarkly/client.go

```diff
@@ -0,0 +1,79 @@
+package launchdarkly
+
+import (
+	"net/http"
```

Comment:
This list needs a `go fmt`, it is not in the correct order.

Reply:
Done. Funny that IntelliJ doesn't do this on his own, though.